FBRetainCycleDetector/Layout/Classes/FBClassStrongLayout.mm

```diff
@@ -89,7 +105,7 @@
     FBIvarReference *wrapper = [[FBIvarReference alloc] initWithIvar:ivar];
 
     if (wrapper.type == FBStructType) {
-      NSString *encoding = @(ivar_getTypeEncoding(wrapper.ivar));
+      std::string encoding = std::string(ivar_getTypeEncoding(wrapper.ivar));
```

Comment:
Consider making this the lone non-reference variable of the encoding string, to avoid repeated copying. Pretty much all other variables of the encoding string can be const references to this one.